### ISSUE TITLE
Request Microsoft Edge compatibility

### ISSUE THREAD

**Issue by darkskygit** [snyderp/web-api-manager#60]
I used the official Microsoft porting tool(https://www.microsoft.com/store/p/microsoft-edge-extension-toolkit/9nblggh4txvb) to port the chrome version to the Edge and it looks like it can run, but in fact I get an error "Unable to find Web API Manager settings: xxxxx".

**Comment by snyderp** [comment]
Interesting, I'm game to port to Edge, but I dont have a machine I can use to test with.  Does MS still publish VM images that developers can use for this sort of purpose?  If so, I can look into moving this over.  It shouldn't be too much lift (famous last words… :P)

**Comment by darkskygit** [comment]
Microsoft provides a virtual machine with a development environment:
https://developer.microsoft.com/en-us/windows/downloads/virtual-machines

**Comment by snyderp** [comment]
I looked into this a bit, but porting it over would take a bit of work.  Edge doesn't have support for a couple of API's the extension needs (the biggest issue is that the the browser action API is very different, but there are a smattering of other things missing too).

I tried running it with their [bridging / porting shim](https://docs.microsoft.com/en-us/microsoft-edge/extensions/guides/porting-chrome-extensions) but the API doesn't seem to be fully compatible.

It would be nice to have this working, but I don't think maintaining an Edge port is something I've got the bandwidth to do.  If someone on Windows wanted to manage it, I'd *happily* accept a PR, but in the meantime, i'm going to mark this as wont fix.

Thank you again for the nudge, and I'll keep an eye on the state of Edge WebExtension support!
